src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/EnclaveTest.java

```diff
@@ -206,656 +180,172 @@ public void testAEv2Disabled(String serverName, String url, String protocol) thr
 
         try (SQLServerConnection con = PrepUtil.getConnection(testConnectionString);
                 SQLServerStatement stmt = (SQLServerStatement) con.createStatement()) {
-            String[] values = createCharValues(nullable);
-            testChars(stmt, cekJks, charTable, values, TestCase.NORMAL, true);
+            String[] values = createCharValues(false);
+            TestUtils.dropTableIfExists(CHAR_TABLE_AE, stmt);
```

Comment:
Are these tests tan in parallel or sequentially. If they are ran in parallel, wouldn't there be a conflict because tests are using the same table?

Reply:
The classes themselves aren't run in parallel, e.g. EnclaveTest can run alongside JDBCEncryptionDecryption, but multiple EnclaveTest test cases can't be run at the same time. I don't think the chances of conflicts are very high but we can circumvent it either way by forcing the tests to use/create different databases.